lib/libutils/ext/include/util.h

```diff
@@ -135,7 +135,7 @@
  * GENMASK_64(39, 21) gives us the 64bit vector 0x000000ffffe00000.
  */
 #define GENMASK_32(h, l) \
-	(((~UINT32_C(0)) << (l)) & (~UINT32_C(0) >> (32 - 1 - (h))))
+	(((~UINT32_C(0)) << (l)) & ~((~UINT32_C(0) << 1) << (h)))
```

Comment:
The reason of shifting in two steps is to avoid `error: left shift count >= width of type [-Werror=shift-count-overflow]`.

Good suggestion! I'll update the macro to `((UINT32_C(0xffffffff) << (l)) & (UINT32_C(0xffffffff) >> (32 - 1 - (h))))`. Thank you.

Reply:
Updated and rebased on the latest `master`.